oarepo_oidc_einfra/config.py

```diff
@@ -46,9 +54,49 @@
 # EINFRA_REPOSITORY_FACILITY_ID = 0
 # """Internal ID of the facility in the E-INFRA Perun that represents the repository."""
 
-# EINFRA_CAPABILITIES_ATTRIBUTE_ID = 0
-# """Internal ID of the attribute in the E-INFRA Perun that represents the capabilities."""
+EINFRA_CAPABILITIES_ATTRIBUTE_NAME = "urn:perun:resource:attribute-def:def:capabilities"
+"""urn of the attribute in the E-INFRA Perun that represents the capabilities."""
 
-# EINFRA_SYNC_SERVICE_ID = 0
-# """Internal ID of the service in the E-INFRA Perun that is responsible for synchronization
+# EINFRA_SYNC_SERVICE_NAME = "..."
+# """name of the service in the E-INFRA Perun that is responsible for synchronization
 # (creating and pushing dumps with resources and users)."""
+
+# EINFRA_USER_DUMP_S3_ACCESS_KEY = ""
+# """Access key for the S3 bucket where the user dump from PERUN is stored."""
+#
+# EINFRA_USER_DUMP_S3_SECRET_KEY = ""
+# """Secret key for the S3 bucket where the user dump from PERUN is stored."""
+#
+# EINFRA_USER_DUMP_S3_ENDPOINT = ""
+# """Endpoint for the S3 bucket where the user dump from PERUN is stored."""
+#
+# EINFRA_USER_DUMP_S3_BUCKET = ""
+# """Bucket where the user dump from PERUN is stored."""
+
+EINFRA_USER_EINFRAID_ATTRIBUTE = (
```

Comment:
EINFRA_ID to keep consistent naming with 81 :)

Reply:
Btw why is this needed when we have: `EINFRA_USER_PERSISTENT_EINFRA_ID_ATTRIBUTE` is there any difference?

Reply:
Perun DUMP contains only the virtual, computed attribute (EINFRA_USER_PERSISTENT_EINFRA_ID_ATTRIBUTE=urn:perun:user:attribute-def:virt:login-namespace:einfraid-persistent). Searching by this attribute is very slow as it does linear search. That's why the recommendation from perun folks is to use the shadow attribute for search instead (EINFRA_USER_EINFRA_ID_ATTRIBUTE=urn:perun:user:attribute-def:def:login-namespace:einfraid-persistent-shadow)

Reply:
renaming: done